src/ResourceManager/Network/Commands.Network/VirtualNetworkGatewayConnection/NewAzureVirtualNetworkGatewayConnectionCommand.cs

```diff
@@ -130,6 +131,18 @@ public class NewAzureVirtualNetworkGatewayConnectionCommand : VirtualNetworkGate
             HelpMessage = "Do not ask for confirmation if you want to overrite a resource")]
         public SwitchParameter Force { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Whether to enable policy-based traffic selectors for a S2S connection")]
+        public string UsePolicyBasedTrafficSelectors { get; set; }
```

Comment:
@henry416 oops, sorry, I thought the two new parameters should be grouped together. In that case, you should just make this `string` parameter into a `SwitchParameter` called `EnablePolicyBasedTrafficSelectors`.

Following this pattern will make for a much better user experience - if the user wants to enable policy-based traffic selectors, then they will add `-EnablePolicyBasedTrafficSelectors` to the end of the command, and if they don't want to use it, then they don't have to do anything extra. Having the user provide a `string` that is being parsed into a `bool` is a less user-friendly way of having using a `SwitchParameter`.